4-hangman/hangman02.py

```diff
@@ -0,0 +1,13 @@
+import random
+my_list = ["I", "eat", "chicken", "and", "turkey"]
+random_word = str(random.choice(my_list))
+# print(random_word.upper())
+
+word = input("Kindly type one letter only: ")
+index = 0
+while index < len(random_word):
+    if word in random_word[index]:
```

Comment:
Think of what you want to do here. Why are you using 'in'?

Reply:
While iterating through each word, i wrote if (for example word is l) l in random_word[0] hmmmm 
so i can use == instead of in. Thanks